Envelopes/saull_Peak Envelope Generator.lua

```diff
@@ -0,0 +1,277 @@
+-- @description Peak Envelope Generator
+-- @author saul-l
+-- @version 1.0
+-- @about
+--   # Peak envelope generator
+--
+--   Generates envelope based on audio item peaks.
+--   You can think of it as an offline audio source parameter baker.
+--
+--   Requires ReaImGui, but will prompt you to install it and provide ReaPack repo,
+--   if you don't already have it.
+--
+--   Contains built-in documentation in UI.
+--
+--   Only works with FX envelopes at the moment
+
+
+
+
+if not reaper.ImGui_GetBuiltinPath then
+  reaper.MB("ReaImGui required. Please install it.", "Install ReaImGui", 0)
+  if not pcall(function() reaper.ReaPack_BrowsePackages( "ReaImGui: ReaScript binding for Dear ImGui" ) end) then
+    reaper.MB("Attempting to open ReaPack failed. Visit https://reapack.com/", "No ReaPack found", 0)
+    if reaper.GetOS() == "Win32" or reaper.GetOS() == "Win64" then
+      os.execute('start https://reapack.com')
+    else
+      os.execute('open https://reapack.com')
```

Comment:
On Linux it's `xdg-open`.

(There's `CF_ShellExecute('url')` in SWS that's cross-platform & avoids using the system shell but I don't think it would be appropriate here since it'd be yet another dependency that could be missing.)

(`ReaPack_BrowsePackages` itself won't emit any errors, but a simpler presence check could be `if not ReaPack_BrowsePackages then`.)